### ISSUE TITLE
Rado stops playing if there's a new one

### ISSUE THREAD

**Issue by SpeeterYT** [slowlydev/f1-dash#141]
If I am listening to a team radio message and a new one becomes available playback stops. Tested on chrome and firefox

**Comment by slowlydev** [comment]
Yes I also have noticed that. Is either or a combination between framer-motion, the react keys on the map children, or the map in general.
